src/components/notifications/SevenIO.vue

```diff
@@ -0,0 +1,22 @@
+<template>
+    <div class="mb-3">
+        <label for="sevenio-api-key" class="form-label">{{ $t("apiKeySevenIO") }}</label>
+        <input id="sevenio-api-key" v-model="$parent.notification.sevenioApiKey" type="text" class="form-control" required autocomplete="false">
```

Comment:
Please use `HiddenInput` for sensitive fields instead ^^

Reply:
@scolastico you did mark this resolved hat this is not implemented. Could you change the input for the `HiddenInput`?